plan/physical_plans.go

```diff
@@ -127,6 +143,15 @@ func (p *PhysicalIndexLookUpReader) Copy() PhysicalPlan {
 	return &np
 }
 
+// MarshalJSON implements json.Marshaler interface.
+func (p *PhysicalIndexLookUpReader) MarshalJSON() ([]byte, error) {
+	buffer := bytes.NewBufferString("{")
+	buffer.WriteString(fmt.Sprintf("\"read index from\":\"%s\",", p.indexPlan.ID()))
```

Comment:
for `PhysicalIndexLookUpReader`, it's the data source of root task and have no children

It will read index from cop task, i.e its indexPlan, so I print the message "read index from"